manifests/script-lab-local.xml

```diff
@@ -42,6 +42,16 @@
     <Description resid="PG.Description" />
     <Hosts>
       <Host xsi:type="Workbook">
+        <AllFormFactors>
+          <ExtensionPoint xsi:type="CustomFunctions">
+            <Script>
+              <SourceLocation resid="customfunctionsjs" />
```

Comment:
Why do you need to specify both html and JS file?  Should the HTML just load the right JS file(s)?  Just like add-incommands do?